docker/multiplexer.Dockerfile

```diff
@@ -15,7 +15,7 @@ ARG MAX_SQUARE_SIZE
 # docker build --build-arg UPGRADE_HEIGHT_DELAY=1000 -t celestia-app:latest .
 ARG UPGRADE_HEIGHT_DELAY
 # the docker registry used for the embedded v3 binary.
-ARG CELESTIA_APP_REPOSITORY=ghcr.io/celestiaorg/celestia-app
+ARG CELESTIA_APP_REPOSITORY=ghcr.io/celestiaorg/celestia-app-standalone
 ARG CELESTIA_VERSION="v3.10.0-rc0"
```

Comment:
this will actually break as the image does not exist in this registry. We'll need to also update the v3 line to publish images to the `ghcr.io/celestiaorg/celestia-app-standalone` registry.

Reply:
#4719